.changeset/shy-carpets-add.md

```diff
@@ -0,0 +1,7 @@
+---
+'@getodk/scenario': patch
+'@getodk/xpath': patch
+'@getodk/web-forms': patch
+---
+
+Throw an error when jr:choice_name is called with an element reference that does exist
```

Comment:
There’s a missing “not”! Small nit since I’m here: how about “reference” rather than “element reference”?

Reply:
Thanks! Fixed.